src/dbxincluder/docbook.py

```diff
@@ -143,18 +143,19 @@ def fixup_references(subtree):
             elem.set(attr, " ".join(new_targets))
 
 
-def process_tree(tree, base_url, file):
+def process_tree(tree, base_url, xmlcatalog, file):
```

Comment:
Oh, just caught this: maybe use also None as default (as stated in the docstring)?